workspaces/github-actions/plugins/github-actions/src/alpha/entityContent.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { screen, waitFor } from '@testing-library/react';
+import { createExtensionTester } from '@backstage/frontend-test-utils';
+import * as cards from './entityCards';
+import {
+  createApiExtension,
+  createApiFactory,
+} from '@backstage/frontend-plugin-api';
+import { GithubActionsApi, githubActionsApiRef } from '../api';
+import { sampleEntity } from '../__fixtures__/entity';
+
+jest.mock('@backstage/plugin-catalog-react', () => ({
+  ...jest.requireActual('@backstage/plugin-catalog-react'),
+  useEntity: () => sampleEntity,
+}));
+
+jest.mock('@backstage/core-plugin-api', () => ({
+  ...jest.requireActual('@backstage/core-plugin-api'),
+  useRouteRef: () => () => '/github-actions',
+}));
+
+const listBranchesMock = jest
+  .fn()
+  .mockReturnValue([])
+  .mockReturnValueOnce(require('../__fixtures__/list-branches.json'));
+
+const listWorkflowRunsMock = jest
+  .fn()
+  .mockReturnValue(require('../__fixtures__/list-workflow-runs.json'));
```

Comment:
Just wanna note that we'll be revising `createExtensionTester` a bit to make it easier to render a test subject within a React tree, rather than it aways being at the root. That should make it possible to easily get rid of these mocks through `TestApiProvider`, `EntityProvider`, and mocked routes.